cypress/integration/move.spec.js

```diff
@@ -874,147 +634,97 @@ describe("Move flow", () => {
     const TITLE_RESOURCE_PAGE = "Move resource page"
 
     beforeEach(() => {
-      cy.setCookie(COOKIE_NAME, COOKIE_VALUE)
-      window.localStorage.setItem("userId", "test")
       cy.visit(
         `${CMS_BASEURL}/sites/${TEST_REPO_NAME}/resourceRoom/${PARSED_TEST_REPO_RESOURCE_ROOM_NAME}/resourceCategory/${PARSED_TEST_REPO_RESOURCE_CATEGORY_NAME}`
       )
-      waitForDom()
+      cy.contains("Verify").should("not.exist")
     })
 
     it("Should be able to navigate from Resource Category to Resource Room back to Resource Category via MoveModal buttons", () => {
-      cy.contains(TITLE_RESOURCE_PAGE).should("exist")
-      cy.get(`button[id^="pageCard-dropdown-"]`).should("exist").click()
-      waitForDom()
-      cy.get("button[id^=move-]").first().trigger("mousedown")
-      cy.contains(`Move Here`)
+      cy.contains("a", TITLE_RESOURCE_PAGE).as("resourcePage").should("exist")
+      cy.clickContextMenuItem("@resourcePage", "Move to")
+      cy.contains("button", "Move Here")
 
       // Assert
```

Comment:
Resource tests are currently failling - this is due to the different resource names. Will ignore as the resource names will be fixed in the respective PR